docker/dev.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:10-stretch
+
+# Allow exposing HTTP endpoint
+EXPOSE 9000
+
+# Add project source
+COPY . /code
```

Comment:
any reason to copy everything if we map it through docker-compose anyway? (I'd assume that we don't use that image outside of docker-compose)